tests/unittests/util.py

```diff
@@ -153,7 +153,7 @@ def update_package_sources(self):
         return (True, "yay")
 
     def do_as(self, command, args=None, **kwargs):
-        return ("stdout", "stderr")
+        return subp.SubpResult("pip 23.0.1 from /usr/lib/python3/dist-packages/pip (python 3.11)", "stderr")
```

Comment:
Feels a little funny shoving test-specific data into the distro mock. Suggestions welcome.